Source/Core/Core/ConfigManager.cpp

```diff
@@ -54,6 +54,7 @@
 #include "Core/TitleDatabase.h"
 #include "VideoCommon/HiresTextures.h"
 
+#include "Core/WC24PatchEngine.h"
```

Comment:
This include should go into the block with the other Core includes. Maybe also add a newline between the Core block and the VideoCommon one while you're there.